lib/pause_2017/PAUSE/Web/Context.pm

```diff
@@ -40,6 +41,17 @@ sub version {
   $version;
 }
 
+sub authenticator_for {
+  my ($self, $user) = @_;
+  my $cpan_alias = lc($user->{userid}) . '@cpan.org';
+  my $secret32   = $user->{mfa_secret32};
```

Comment:
A comment here explaining that `$user->{mfa_secret32}` will be undef the first time this is set up, and that Auth::GoogleAuth will generate one for us, would be nice I think.

Reply:
The generated secret32 at https://metacpan.org/dist/Auth-GoogleAuth/source/lib/Auth/GoogleAuth.pm#L21-25 might not be strong enough. Is it better for us to generate it by ourselves using Crypt::URandom instead of just adding a comment?